test/e2e/lifecycle/bootstrap/util.go

```diff
@@ -81,7 +81,7 @@ func addSecretExpiration(s *v1.Secret, expiration string) {
 // TimeStringFromNow returns the time as a string from now.
 // e.g: 2019-12-03T14:30:40+08:00.
 func TimeStringFromNow(delta time.Duration) string {
-	return time.Now().Add(delta).Format(time.RFC3339)
+	return time.Now().Add(delta).UTC().Format(time.RFC3339)
```

Comment:
https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/test/e2e/lifecycle/bootstrap/bootstrap_token_cleaner.go#L76

It will be used here